web/crux/src/app/node/node.service.ts

```diff
@@ -175,19 +176,79 @@ export default class NodeService {
     return await this.agentService.getNodeEventsByTeam(request.id)
   }
 
-  handleWatchContainerStatus(request: WatchContainerStateRequest): Observable<ContainerStateListMessage> {
+  async handleWatchContainerStatus(
+    request: WatchContainerStateRequest,
+  ): Promise<Observable<ContainerStateListMessage>> {
     this.logger.debug(`Opening container status channel for prefix: ${request.nodeId} - ${request.prefix}`)
 
+    const defaultImages = await this.prisma.deployment.findUniqueOrThrow({
+      where: {
+        id: request.deploymentId,
+      },
+      include: {
+        instances: {
+          include: {
+            image: {
+              select: {
+                name: true,
+              },
+            },
+          },
+        },
+      },
+    })
+
+    const defaultContainerState = {
+      prefix: request.prefix,
+      data: defaultImages.instances.map(
+        it =>
+          ({
+            name: `/${request.prefix}-${it.image.name}`,
```

Comment:
Afaik the container name is `image.config.name`

Reply:
They are the same. Maybe its redundant?

Reply:
Not sure then 🤷‍♀️